core/src/main/java/nl/nn/adapterframework/util/flow/DefaultFlowGenerator.java

```diff
@@ -82,6 +86,10 @@ public void destroy() {
 		if(log.isTraceEnabled()) log.trace("destroyed V8FlowEngine");
```

Comment:
Je moet hier, en op andere plekken in de logging ook de referentie naar V8 verwijderen

Reply:
potverdikke, je hebt gelijk